packages/adapter-components/src/adapter-wrapper/adapter/adapter.ts

```diff
@@ -145,6 +146,13 @@ export class AdapterImpl<
     this.changeValidators = {
       ...getDefaultChangeValidators(),
       ...(this.definitions.deploy?.instances === undefined ? { deployNotSupported: deployNotSupportedValidator } : {}),
+      ...(this.definitions.deploy !== undefined
+        ? {
+            createCheckDeploymentBasedOnDefinitions: createCheckDeploymentBasedOnDefinitionsValidator({
+              deployDefinitions: this.definitions.deploy,
+            }),
+          }
+        : {}),
```

Comment:
can't this be
```suggestion
      ...(this.definitions.deploy?.instances === undefined
      ? {
        deployNotSupported: deployNotSupportedValidator.
      } : {
        createCheckDeploymentBasedOnDefinitions: createCheckDeploymentBasedOnDefinitionsValidator({
              deployDefinitions: this.definitions.deploy,
            }),
        }),
```
(up to indentation etc)?

Reply:
of course